cmd/import.go

```diff
@@ -55,7 +55,7 @@ omitted. If it is present then its format should be YYYY-MM-DDTHH:MM.
 	flags.StringVarP(&Importer.Host, "host", "", "localhost:10101", "host:port of Pilosa.")
 	flags.StringVarP(&Importer.Index, "index", "i", "", "Pilosa index to import into.")
 	flags.StringVarP(&Importer.Frame, "frame", "f", "", "Frame to import into.")
-	flags.StringVarP(&Importer.Field, "field", "", "", "Field to import into.")
+	flags.StringVarP(&Importer.BSIGroup, "bsi-group", "", "", "BSIGroup to import into.")
```

Comment:
ugh, "bsi group" does not make for a very friendly term in the UI... we'll have to think of something to make this more approachable

Reply:
well, i'm working to make `bsiGroup` un-exported, so we really shouldn't be using it anywhere externally.

but I agree, `bsiGroup` was not the really the best choice of terms given that it makes for awkward names and comments.